deployment/cloud/main.tf

```diff
@@ -0,0 +1,47 @@
+terraform {
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "~> 5.61"
+    }
+  }
+}
+
+provider "aws" {
+  region = "us-west-2"
+  assume_role {
+    role_arn = "arn:aws:iam::871683513797:role/supportsphere-deploy"
+    session_name = "supportsphere-infra-deployment"
+    external_id = "supportsphere-infra-deployment"
+  }
+
+  // Tags all resources created from this provider with {"Project": "Support Sphere", "Neighborhood": "Laurelhurst"}
+  default_tags {
+    tags = {
+      Project = "Support Sphere",
+      Neighborhood = "Laurelhurst"
```

Comment:
for D.R.Y purposes, can these values be put into the variables? Seems like this is hardcoded in a few places.

Reply:
absolutely, you're totally right -- can do :)

Reply:
done in latest revision :)